app/views/interstitial/ViewNationalInsuranceInterstitialHomeView.scala.html

```diff
@@ -67,6 +67,6 @@
         </div>
     } else {
         @p(Text(messages("label.you_can_see_this_part_of_your_account_if_you_complete")))
-        @button(href = Some(routes.ApplicationController.uplift(Some(SafeRedirectUrl(redirectUrl))).url), id="continue", msg = "global.label.continue_button")
+        @button(href = Some(routes.ApplicationController.uplift(Some(RedirectUrl(redirectUrl))).url), id="continue", msg = "global.label.continue_button")
```

Comment:
The import needs to be updated to @import uk.gov.hmrc.play.bootstrap.binders.RedirectUrl